cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -162,8 +151,8 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
 
         val config = globalOptionsForSubcommands.config
         val analyzerConfig = AnalyzerConfiguration(
-            ignoreToolVersions,
-            allowDynamicVersions,
+            config.analyzer?.ignoreToolVersions == true,
+            config.analyzer?.allowDynamicVersions == true,
             config.analyzer?.sw360Configuration
         )
         val analyzer = Analyzer(analyzerConfig)
```

Comment:
I believe this can now be simplified to just

    val analyzer = Analyzer(config.analyzer ?: AnalyzerConfiguration())